Projects/Movie_Recommendation.py

```diff
@@ -0,0 +1,103 @@
+# Python3 code for movie recommendation based on your emotion
+
+# Import library for web
+# scrapping
+from bs4 import BeautifulSoup as SOUP
+import re
+import requests as HTTP
+
+# Main Function for scraping
+
+
+def main(emotion):
+
+    # IMDb Url for Drama genre of
+    # movie against emotion Sad
+    if (emotion == 6):
+        urlhere = 'http://www.imdb.com/search/title?genres=drama&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Musical genre of
+    # movie against emotion Disgust
+    elif (emotion == 3):
+        urlhere = 'http://www.imdb.com/search/title?genres=musical&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Family genre of
+    # movie against emotion Anger
+    elif (emotion == 1):
+        urlhere = 'http://www.imdb.com/search/title?genres=family&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Thriller genre of
+    # movie against emotion Anticipation
+    elif (emotion == 2):
+        urlhere = 'http://www.imdb.com/search/title?genres=thriller&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Sport genre of
+    # movie against emotion Fear
+    elif (emotion == 4):
+        urlhere = 'http://www.imdb.com/search/title?genres=sport&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Thriller genre of
+    # movie against emotion Joy
+    elif (emotion == 5):
+        urlhere = 'http://www.imdb.com/search/title?genres=thriller&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Western genre of
+    # movie against emotion Trust
+    elif (emotion == 8):
+        urlhere = 'http://www.imdb.com/search/title?genres=western&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Film_noir genre of
+    # movie against emotion Surprise
+    elif (emotion == 7):
+        urlhere = 'http://www.imdb.com/search/title?genres=film_noir&title_type=feature&sort=moviemeter, asc'
+
```

Comment:
Missing check if user enters wrong input

Reply:
No need for so many `if - elif` statements.
Make a `dict` store serial number and genres and make URL
Use that URL to make the request.

If the serial number doesn't exist in `dict`, you can add a wrong input warning!